entity-framework/core/what-is-new/ef-core-5.0/whatsnew.md

```diff
@@ -142,6 +142,20 @@ Previously when scaffolding a DbContext from an existing database, EF Core will
 
 In previous releases, this behavior was configurable through registration of a pluralization service. Now in EF Core 5.0, the [Humanizer](https://www.nuget.org/packages/Humanizer.Core/) package is used as a default pluralization service. This means tables `People` and `Addresses` will now be reverse engineered to entity types named `Person` and `Address`.
 
+### Savepoints
+
+EF Core now supports [savepoints](https://en.wikipedia.org/wiki/Savepoint) for greater control over transactions that execute multiple operations.
```

Comment:
Is Wikipedia the best link for this? I'm not against it, just wasn't sure if there is something in our docs we can link to.

Reply:
Open to suggestions...

Reply:
Does [this one](https://docs.microsoft.com/SQL/t-sql/language-elements/save-transaction-transact-sql#remarks) work?

Reply:
It's SQL Server specific, but I think it will do.